MaterialLib/MPL/mpMedium.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * \author Norbert Grunwald
+ * \date   07.09.2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "BaseLib/ConfigTree.h"
+#include "mpPhase.h"
+#include "mpProperty.h"
+
+#include <memory>
+#include <vector>
+
+namespace MaterialPropertyLib
+{
+/**
+ * \class Medium
+ * \brief This class is for material objects on the Medium scale.
+ * \details A Medium consists of an arbitrarily long vector of phases
+ * and an array of properties.
+ */
+class Medium
```

Comment:
The class name is different from the file name. Can you explain the meaning of the file name of mpMedium?